lib/ezapi_client/services/exec_command.rb

```diff
@@ -4,18 +4,23 @@ class ExecCommand
     include Virtus.model
     attribute :command, String
 
-    def self.call(command)
-      self.new(command: command).()
+    def self.call(command, logger = nil)
+      if logger
+        self.new(command: command).(logger)
+      else
+        self.new(command: command).()
```

Comment:
Oh you can always pass logger. No need for this since you already have the `if` in line 23

Reply:
ah yes! thanks for the catch, I added the if on line 23 on a later time.